abel/tools/circularize.py

```diff
@@ -0,0 +1,320 @@
+# -*- coding: utf-8 -*-
+from __future__ import division
+import numpy as np
+from scipy import ndimage
+from scipy.interpolate import UnivariateSpline
+from scipy.optimize import leastsq
+
+import abel
+
+#########################################################################
+# circularize.py
+#
+# Image circularization by following peak intensity vs angle
+# see https://github.com/PyAbel/PyAbel/issues/186 for discussion
+# and https://github.com/PyAbel/PyAbel/pull/195
+#
+# Steve Gibson and Dan Hickstein - ideas/code
+# Jason Gascooke - ideas
+#
+# February 2017
+#########################################################################
+
+
+def circularize_image(IM, method="lsq", center=None, radial_range=None,
+                      dr=0.5, dt=0.5, smooth=0, ref_angle=None,
+                      inverse=False, return_correction=False):
+    """
+    Corrects image distortion on the basis that the structure (Newton spheres)
```

Comment:
Can you move the mention of "Newton Spheres" to the next paragraph, in order to keep the first paragraph general, and start into the specifics of VMI in the next paragraph?